subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/TaskReportTask.java

```diff
@@ -66,12 +67,22 @@ public boolean isDetail() {
         return detail;
     }
 
+    @Option(option = "group", description = "Show tasks for a specific group.")
+    public void setDisplayGroup(String group) {
+        this.group = group;
+    }
+
+    @Console
+    public String getDisplayGroup() {
```

Comment:
Please add the `@Incubating` annotation and `@since 5.1` to the Javadoc (that's why Travis failed).